optaweb-employee-rostering-standalone/README.adoc

```diff
@@ -0,0 +1,48 @@
+[[optaweb-employee-rostering-standalone]]
+= OptaWeb Employee Rostering Standalone
+
+== Build and Run with Maven
+
+In the project backend directory, run the following command:
```

Comment:
For that exact command in backend directory, it will repack the jars. Surely user is responsible to provide thouse jars. But you are right in the beginning they are not there, so it is better to mention that in the first place the user needs to build the root module.

Reply:
Ok, I see I used the wrong name of the folder - I meant standalone folder. Ok will fix that